test/custom_error.ts

```diff
@@ -0,0 +1,14 @@
+/**
```

Comment:
it is used here.
https://github.com/JiaLiPassion/zone.js/blob/f318c3b1fce222f0d4551555ca5834b37e4868d6/test/common/Error.spec.ts#L167